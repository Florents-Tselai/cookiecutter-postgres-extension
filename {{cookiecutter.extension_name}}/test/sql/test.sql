select add_one(1);

select copytext('hello world');

select concat_text('pa', 'sok');
