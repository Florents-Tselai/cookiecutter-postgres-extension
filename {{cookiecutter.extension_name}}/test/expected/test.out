select add_one(1);
 add_one 
---------
       2
(1 row)

select copytext('hello world');
  copytext   
-------------
 hello world
(1 row)

select concat_text('pa', 'sok');
 concat_text 
-------------
 pasok
(1 row)

